CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 [Full Changelog](https://github.com/ParaConUK/advtraj/compare/v0.4.0...HEAD)
 
+*Support for MONC*
+Primarily changes to support the MONC grid. This has:
```

Comment:
I think it would be clearer to put this in `advtraj.utils.grid` so we detail in there how the MONC grid works with coordinates on cell-faces. Also, I'm not sure what you mean by "p point" here. Does that refer to points that are for scalar values? Sorry if this is c-grid jargon that I'm just unaware of / have forgotten.

For the changelog I would just write "add support for MONC grid by implementing its c-grid configuration for where the position scalars are stored" or something like that, assuming that is correct.